### PR TITLE
ID-79

### PR DESCRIPTION
**Pull request by xiangzhe0812**
onboarding page: once user  submit the form then it will goto the main
WechatIMG12

Screen Shot 2021-08-04 at 3 53 07 pm

### CODE REVIEW

**Review by rellfy — APPROVED** [review]

**Comment by bitcoinbrisbane** [comment]
No need to have `...` email and mobile

File: app/screens/claim/DateClaim.tsx [hunk 1]
```diff
@@ -27,7 +27,8 @@ const DateClaim = ({
         }}
       >
         <Text style={{ color: item.value ? undefined : colors.gray }}>{`${
-          moment(item.value).format("DD/MM/YYYY") || "Please enter the specific date..."
+          moment(item.value).format("DD/MM/YYYY") ||
+          "Please enter the specific date..."
```
Comment (bitcoinbrisbane) [inline app/screens/claim/DateClaim.tsx:31]:
```suggestion
          "Please enter the specific date"
```

File: app/screens/onboard/index.tsx [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+                onBlur={handleBlur("name")}
+              />
+              {errors.name && (
+                <Text style={styles.baseStyle.errorMessage}>{errors.name}</Text>
+              )}
+              <Text style={styles.baseStyle.baseText}>Email</Text>
+              <TextInput
+                style={styles.baseStyle.baseTextInput}
+                keyboardType="email-address"
+                placeholder="Please enter your email address..."
```
Comment (bitcoinbrisbane) [inline app/screens/onboard/index.tsx]:
```suggestion
                placeholder="Please enter your email address"
```

File: server/src/endpoint/claims/verify-claim.ts [hunk 3]
```diff
@@ -5,18 +5,11 @@ import { validationBadRequest } from "../../utils/errors";
...
-  streetName: Joi.string().required(),
-  streetType: Joi.string().required(),
-  locality: Joi.string().required(),
-  region: Joi.string().required(),
-  postCode: Joi.string().required(),
-  country: Joi.string().required()
+  name: Joi.string().empty(""),
+  email: Joi.string().empty(""),
+  address: Joi.string().empty(""),
+  mobile: Joi.string().empty("")
```
Comment (bitcoinbrisbane) [inline server/src/endpoint/claims/verify-claim.ts]:
```suggestion
  address: Joi.string().empty(""),
  email: Joi.string().empty(""),
  mobile: Joi.string().empty(""),
  name: Joi.string().empty("")
```

**Review by bitcoinbrisbane — APPROVED** [review]
LGTM

**Pull request merged**